src/IdBrokerClient.php

```diff
@@ -230,4 +285,41 @@ public function updateUser(array $config = [])
         
         $this->reportUnexpectedResponse($result, 1490808841);
     }
+
+    /**
+     * Determine whether any of the Id-broker's IPs are not in the
+     * trusted ranges
+     *
+     * @throws Exception
+     */
+    private function checkIdBrokerIP() {
```

Comment:
This function should probably have a name starting with `assert`, maybe `assertTrustedBrokerIp` which communicates better that it throws an exception if not valid.